lib/spack/spack/package.py

```diff
@@ -1053,6 +1053,9 @@ def do_fake_install(self):
         touch(join_path(self.prefix.bin, 'fake'))
         mkdirp(self.prefix.include)
         mkdirp(self.prefix.lib)
+        library_name = 'lib' + self.name
+        touch(join_path(self.prefix.lib, library_name + '.so'))
```

Comment:
@davydden Thanks. FYI, the `fake_*` are functions that fake installations for debug purposes. If you ever need for instance to check just the DB entry for `dealii` you can do:
```console
spack install --fake dealii
```
and you will skip the installation of the packages. It's also useful if you need to show a live demo and don't want to wait forever on a build.